dashboard/app/views/callouts/show.json.jbuilder

```diff
@@ -1 +0,0 @@
-json.extract! @callout, :element_id, :text, :created_at, :updated_at
```

Comment:
Note that `callout.text` does not exist, so hitting (e.g.) studio.code.org/callouts/1.json gives a 500.

https://app.honeybadger.io/projects/3240/faults/32645619